lib/assessment.js

```diff
@@ -24,7 +28,13 @@ var sum = function sum() {
 // if invoking the callback for any element
 // in the array returns a truthy value
 var any = function any(array, callback, thisArg) {
+  var result = true;
 
+  array.forEach(function(currentValue, index, array) {
+    result = callback(currentValue, index, array);
+  })
```

Comment:
Not quite. We want this function to give us back a `true` if any element is true; as written, it will only give us the value (true or false) of the last element.

Can you figure out what this would need in order to be correct?
(HINT: Try using `||`).